src/Export.as

```diff
@@ -0,0 +1,5 @@
+namespace PluginManager
+{
+    import void PluginUninstall(Meta::Plugin@ plugin) from "PluginManager";
```

Comment:
We should document these with comments; take a look at `VehicleState` for example. Important to note here is that the provided `plugin` handle will become invalid after calling it.